Libraries/Opc.Ua.PubSub/Encoding/JsonNetworkMessage.cs

```diff
@@ -179,10 +178,7 @@ public void SetNetworkMessageContentMask(JsonNetworkMessageContentMask networkMe
         /// <returns></returns>
         public override byte[] Encode()
         {
-            IServiceMessageContext messageContext = new ServiceMessageContext {
-                NamespaceUris = ServiceMessageContext.GlobalContext.NamespaceUris,
```

Comment:
to directly use the GlobalContext here seems like a hack

Reply:
Ok, we will have to refactor this method and pass the messageContext as a parameter.
We recall that you requested that an Encode method should exist without any parameters, is it ok in your opinion that we refactor it and add the IServiceMessageContext parameter?

Reply:
yes, that would make sense.